wai-aria/role/contextual-roles.tentative.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+<head>
+  <title>Tentative: Contextual Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<!--
+  New sectionheader and sectionfooter roles.
+  See https://github.com/w3c/aria/pull/1931
+-->
+<main>
+    <footer data-testname="footer element scoped to main element is sectionfooter" data-expectedrole="sectionfooter" class="ex">x</footer>
+</main>
```

Comment:
This `main > footer` subtest should probably be moved to the same html-aam file as the others above... It may have been here b/c this dir pre-dated the html-aam dir.

This new file in wai-aria could test author-provided role tokens:  role="sectionheader" and `role="sectionfooter"`

Reply:
Updated, thanks!